MainRepository/Program.cs

```diff
@@ -1,7 +1,44 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿// Implementation of Anagram

```

Comment:
Write proper comment about what this program is doing.
This finds all anagrams and sorts them based on anagram count